Dockerfile

```diff
@@ -33,6 +32,7 @@ RUN dpkg --add-architecture armel \
         libseccomp-dev:armhf \
         libseccomp-dev:ppc64el \
         libseccomp2 \
+	npm \
```

Comment:
Looks like this doesn't use the same indentation (tabs vs. spaces) as the surrounding lines.